package.json

```diff
@@ -89,8 +91,10 @@
     "eslint": "^3.19.0",
     "exists-sync": "0.0.3",
     "lodash": "^4.12.0",
-    "lodash.defaultsdeep": "^4.5.1",
-    "mocha-eslint": "^3.0.1"
```

Comment:
`mocha-eslint` was mistakenly in `dependencies`, this just moves it `devDependencies`.